docs/security/authorization-pushdown.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Permission System"
+slug: /security/permission-system
+keyword: security
+license: "This software is licensed under the Apache License version 2."
+---
+
+## Authorization Push-down
+
+![authorization push down](../assets/security/authorization-pushdown.png)
+
+Gravitino also provides a set of authorization frameworks to interact with different underlying data source
+permission systems (e.g., MySQL's own permission management and the Apache Ranger permission management system for big data)
+in accordance with its own authorization model and methodology.
+On top of this, Gravitino manages different underlying data sources through Catalogs.
+When a user performs an authorization operation on the data in a Catalog, Gravitino will call the interface of the Authorization Plugin module in the respective Catalog to translate the Gravitino authorization model into the underlying data source's permission rules of the underlying data source.
+Permission is then pushed down to the underlying permission system through the client of the underlying data source (JDBC or Apache Ranger client, etc.).
+
+### Authorization Ranger Hive Plugin properties
+
+The Authorization Ranger Hive Plugin extends the following properties in the [Apache Hive catalog properties](../apache-hive-catalog.md#catalog-properties):
```

Comment:
> the following properties in the [Apache Hive catalog properties

Have you put the the following configurations to `../apache-hive-catalog.md#catalog-properties`, If not, you may need to modify the description.

Reply:
OK, I added description in the apache-hive-catalog.md